Misc/NEWS.d/next/VxWorks/README.rst

```diff
@@ -0,0 +1,3 @@
+Put news entry `blurb`_ files for the *VxWorks* section in this directory.
+
+.. _blurb: https://pypi.org/project/blurb/
```

Comment:
I would prefer to not add a new blurb category just for VxWorks. Would you mind to reuse the Library category?

Reply:
Fine. Fixed accordingly.